com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/TargetData/LightingData.cs

```diff
@@ -88,5 +88,13 @@ public bool overrideBakedGI
             get => m_OverrideBakedGI;
             set => m_OverrideBakedGI = value;
         }
+
+        [SerializeField]
+        bool m_EmissionOverriden;
```

Comment:
so it is not in lighting but should be in lit data itself (I don't remind if we have such a struct)